app/src/full/AndroidManifest.xml

```diff
@@ -12,9 +12,15 @@
 		<meta-data
 			android:name="com.google.firebase.messaging.default_notification_color"
 			android:resource="@color/color_base" />
-
-		<service android:name=".common.FirebaseMessagingService">
-			<intent-filter>
+		<meta-data
+			android:name="com.google.firebase.messaging.default_notification_channel_id"
+			android:value="@string/General_AppName" />
+		<meta-data
+			android:name="firebase_analytics_collection_deactivated"
+			android:value="true" />
+		<service android:name=".common.FirebaseMessagingService"
+			android:exported="false">
+		<intent-filter>
```

Comment:
What do these changes do?

Reply:
https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService

Reply:
https://stackoverflow.com/questions/39327141/disable-the-analytics-service-for-firebase-cloud-messaging